Makefile

```diff
@@ -1,14 +1,17 @@
 
-SHELL  ?= $(shell command -v bash 2>/dev/null)
-DOCKER ?= $(shell command -v docker 2>/dev/null)
-PANDOC ?= $(shell command -v pandoc 2>/dev/null)
+EPOCH_TEST_COMMIT	:= 78e6667ae2d67aad100b28ee9580b41b7a24e667
+OUTPUT_DIRNAME		?= output/
```

Comment:
so, not in this PR ;-)

On Wed, Jun 1, 2016 at 3:01 PM, W. Trevor King <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/opencontainers/runtime-spec/pull/478#discussion_r65422302>
> :
>
> > @@ -1,14 +1,17 @@
> >
> > -SHELL  ?= $(shell command -v bash 2>/dev/null)
> > -DOCKER ?= $(shell command -v docker 2>/dev/null)
> > -PANDOC ?= $(shell command -v pandoc 2>/dev/null)
> > +EPOCH_TEST_COMMIT	:= 78e6667ae2d67aad100b28ee9580b41b7a24e667
> > +OUTPUT_DIRNAME		?= output/
>
> On Wed, Jun 01, 2016 at 11:58:45AM -0700, Vincent Batts wrote: > @@ -1,14
> +1,17 @@ > > -SHELL ?= $(shell command -v bash 2>/dev/null) > -DOCKER ?=
> $(shell command -v docker 2>/dev/null) > -PANDOC ?= $(shell command -v
> pandoc 2>/dev/null) > +EPOCH_TEST_COMMIT := 78e6667
> <https://github.com/opencontainers/runtime-spec/commit/78e6667ae2d67aad100b28ee9580b41b7a24e667>
> > +OUTPUT_DIRNAME ?= output/ Well, no. They're similar, but not the same.
> Right. I think they should be unified into a single variable, but that
> unification will take a bit of thinking to figure out.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/opencontainers/runtime-spec/pull/478/files/f656de6a567f016eed7f838d491b27097ad60dd9#r65422302>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAEF6ZPvLh0a8XSGyQUmchtY0C_P0iq1ks5qHdcQgaJpZM4Iryg0>
> .
>